src/web/app/components/question-response-panel/question-response-panel.component.ts

```diff
@@ -64,4 +75,39 @@ export class QuestionResponsePanelComponent {
     return false;
   }
 
+  /**
+   * Loads responses for feedback question.
+   */
+  loadQuestionResults(question: FeedbackQuestionModel): void {
+    this.feedbackSessionsService.getFeedbackSessionResults({
+      questionId: question.feedbackQuestion.feedbackQuestionId,
+      courseId: this.session.courseId,
+      feedbackSessionName: this.session.feedbackSessionName,
+      intent: this.intent,
+      key: this.regKey,
+    }).pipe(finalize(() => {
+      question.isLoaded = true;
+      question.isLoading = false;
+      }))
+      .subscribe((sessionResults: SessionResults) => {
+        const responses: QuestionOutput = sessionResults.questions[0];
+        if (responses) {
+          question.feedbackQuestion = responses.feedbackQuestion;
+          question.allResponses = responses.allResponses;
+          question.otherResponses = responses.otherResponses;
+          question.questionStatistics = responses.questionStatistics;
+          question.responsesFromSelf = responses.responsesFromSelf;
+          question.responsesToSelf = responses.responsesToSelf;
+        } else {
+          question.hasResponse = false;
+        }
+      }, (resp: ErrorMessageOutput) => this.statusMessageService.showErrorToast(resp.error.message));
```

Comment:
Current behavior when error occurs, error toast is shown, question is marked as loaded and "No responses received yet" is shown" or similar message is shown. 

<img width="860" alt="Screen Shot 2022-07-31 at 11 28 29 AM" src="https://user-images.githubusercontent.com/60355570/182008606-dc2e8acf-49c7-4254-99e1-429b7ceb45e4.png">
 
Need to make sure the error message persists similar to https://github.com/TEAMMATES/teammates/pull/11863#issuecomment-1167405928.